.github/workflows/tests.yml

```diff
@@ -61,6 +61,11 @@ jobs:
       run: |
         python setup.py test
 
+    - name: Test under uvloop
```

Comment:
I'd parallelize this into a job via a `loop: [asyncio, uvloop]` matrix

Reply:
done